README.md

```diff
@@ -1,14 +1,25 @@
 # Todo List App
 
 Aplikasi to-do list menggunakan React. Antara front-end dan backend terpisah, dengan techstack:
+
 - Frontend
   - React
 - Backend
   - Golang
 
 ## Dependencies
+
 - docker
-TBD
+- pnpm
+- node.js
+  TBD
 
 ## How to run
-TBD
+
+- pnpm install: Gunakan pnpm untuk menginstal semua dependensi proyek.
+- pnpm run dev: Memulai server pengembangan.
+- pnpm run build: Membuat aplikasi untuk produksi.
+- pnpm run preview: Menjalankan build produksi secara lokal.
+- pnpm run lint: Melakukan linting pada kode menggunakan ESLint (jika dikonfigurasi).
+- pnpm run test: Menjalankan tes (jika pengujian dikonfigurasi).
+  TBD
```

Comment:
TBD = `to be decided` = akan diputuskan nanti.
So remove this because we have the steps to run already.